uaclient/apt_news.py

```diff
@@ -88,6 +100,49 @@ def do_selectors_apply(
         if selectors.pro != _is_attached(cfg).is_attached:
             return False
 
+    if selectors.architectures is not None:
+        if system.get_dpkg_arch() not in selectors.architectures:
+            return False
+
+    if selectors.packages is not None:
+        installed_packages = get_installed_packages()
+        installed_packages_names = [
+            package.name for package in installed_packages
+        ]
+        package_matched = False
+
+        for package in selectors.packages:
+            package_name, version_operator, package_version = package
```

Comment:
We should check that there are three items in this list before doing this. We can log something and return False if there are not three items.